client/test/e2e/cash/cash.spec.js

```diff
@@ -80,36 +86,38 @@ describe('Cash Payments Module', function () {
       browser.get(path);
 
       // the browser should be rerouted to the cashboxB page
-      expect(helpers.getCurrentPath()).to.eventually.equal(target);
+      expect(helpers.getCurrentPath()).to.eventually.equal(path);
     });
 
     it('should allow a user to select and deselect a cashbox', function () {
+      // the auxiliary cashbox is the target
```

Comment:
As a follow up to this comment, I think this can be refined on the long term.  We'll discuss a bit more as a team and develop an issue to track this, if changes are necessary.